azure_pipeline.yaml

```diff
@@ -13,7 +13,62 @@ variables:
   action:
 
 stages:
+  - stage: Precheck
+    jobs:
+      - job: Validate
+        pool:
+          vmImage: $(agentPool)
+        steps:
+        - task: AzureKeyVault@1
+          inputs:
+            ConnectedServiceName: azurerm-sandbox
+            keyVaultName: infra-vault-nonprod
+            secretsFilter: 'azure-devops-token'
+            runAsPreJob: false # Azure DevOps Services only
+        - task: Bash@3
+          env:
```

Comment:
why'd you open a new PR? the old one had a comment requesting a display name here as a form of documenting what this is doing

Reply:
Updated both tasks with a displayName, closed other as was spamming it with files changed to testing. It had a stale entry for notCompleted so how I was doing that in that PR was invalid. Felt a new PR was required.

Reply:
just change your current PR, people lose context and explanations on the previous PR are now gone, also frustrates people when they left a review on the previous one and then it's not there on the new one and comments aren't addressed

Reply:
Sure, will close and reopen old with new changes.